core/src/main/scala/com/salesforce/op/stages/impl/preparators/SanityChecker.scala

```diff
@@ -94,15 +94,24 @@ trait SanityCheckerParams extends DerivedFeatureFilterParams {
   final val maxCorrelation = new DoubleParam(
     parent = this, name = "maxCorrelation",
     doc = "Maximum correlation (absolute value) allowed between a feature in the feature vector and the label",
-    isValid = ParamValidators.inRange(lowerBound = 0.0, upperBound = 1.0, lowerInclusive = true, upperInclusive = true)
+    isValid = ParamValidators.inRange(lowerBound = -0.1, upperBound = 1.1, lowerInclusive = true, upperInclusive = true)
```

Comment:
Floating point - I found in some tests that if you just have a check that the corr be > 1.0 doubles mean that some perfectly correlated features are excluded because they have a corr of 1.00000000004 an I wanted to be able to turn the check off effectively without adding another param

Reply:
@Jauntbox ^^^